openedx/core/djangoapps/credit/tasks.py

```diff
@@ -105,15 +106,18 @@ def _get_credit_course_requirement_xblocks(course):  # pylint: disable=invalid-n
     """
     blocks_stack = [course]
     requirements_blocks = []
+    block_order = 0
     while blocks_stack:
         curr_block = blocks_stack.pop()
         children = curr_block.get_children() if curr_block.has_children else []
         if _is_credit_requirement(curr_block):
+            block_order += 1
```

Comment:
It is intentional as the 0th order will be used for the grade and other will be displayed in order of presence in the courseware.